### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by rocboronat** [vanniktech/Emoji#325]
The project is stable enough to use releases instead of snapshots. Let's move to releases :tada:

**Comment by vanniktech** [comment]
Yes and no. @mario wanted to do some contributions. For a list see - https://github.com/mario/Emoji/issues

I know the last release has been super long ago.

**Comment by vanniktech** [comment]
For now use the SNAPSHOT or build the aars locally.
